src/components/generics/vacation-test-forms/my-vacation-requests.tsx

```diff
@@ -96,6 +169,37 @@ const RenderVacationRequests = () => {
     newOpenRows[index] = !newOpenRows[index];
     setOpenEdit(newOpenRows);
   };
+
+  /**
+  * Handle vacation apply button
+  * Sends vacation request to database
+  *
+  * @param createdRequestId id of newly created vacation request
+  */
+  const createVacationRequestStatus = async (createdRequestId: string | undefined) => {
+    if (!person || !person.keycloakId) return;
+    
+    try {
+      const applyApi = Api.getVacationRequestStatusApi(accessToken?.access_token);
+
+      const createdStatus = await applyApi.createVacationRequestStatus({
+        id: createdRequestId!,
```

Comment:
Avoid this assertion by adding a check for createdRequestId to the above conditon checking !person etc

Reply:
Added a check in above condition. Assertion avoided.

```suggestion
        vacationRequestStatus: {
```